components/navbar/Navbar.js

```diff
@@ -115,10 +116,17 @@ export default function Navbar() {
 
       {session && (
         <>
-          <NavLink
-            item={{ name: "Account", url: "/account/onboarding" }}
-            setIsOpen={setIsOpen}
-          />
+          <Link href="/account/onboarding" aria-label="Account">
+            <div>
```

Comment:
Is this `div` needed?

Reply:
I'm not sure why the manage profile navigation link size is bigger than other link? 
![IMG_20231102_235024](https://github.com/EddieHubCommunity/BioDrop/assets/73095526/da7d7a2e-f057-4331-b9ff-f0459cc7ec1e)


It's my suggestion, I think for consistent design we should use same size for all links, ensuring they look visually related and don't create confusion for users.

Reply:
yes without that div link wont work, don't know why

Reply:
> yes without that div link wont work, don't know why

what does this mean? please explain

Reply:
Well I removed the div, It seems to be working now, sorry for the inconvenience, I don't have any idea why it wasn't working before